### ISSUE TITLE
Cannot set searchBox options.cssClasses.input

### ISSUE THREAD

**Issue by salparadise** [algolia/instantsearch.js#775]
I'm getting error when trying to set the options.cssClasses.input on searchBox widget.

```
Uncaught InvalidCharacterError: Failed to execute 'add' on 'DOMTokenList': 
The token provided ('ais-search-box--input form-control') contains HTML space characters, 
which are not valid in tokens.
```

**Comment by vvo** [comment]
Hi! Can you give us the option value you are trying to pass? Thanks

**Comment by salparadise** [comment]
```CoffeeScript
searchBoxWidget = instantsearch.widgets.searchBox(
    container: '.app-search'
    wrapInput: true
    autofocus: true
    cssClasses:
        root: 'input-group'
        input: 'form-control'
)
```

Just noticed that problem also occurs with root class. Other widgets css classes works fine.

**Comment by bobylito** [comment]
Can you provide the associated markup for the container @salparadise ?

**Comment by salparadise** [comment]
It's as simple as:

```HTML
<div class="app-search"></div>
```

I'm using Bootstrap v4.0.0-alpha.2 and instantsearch.js v1.1.2.
As far as I remember there was the same problem with alpha.1 and instantsearch v1.0.

**Comment by bobylito** [comment]
Thanks @salparadise I was asking this because you can use the search box widget either with an existing input field or just a div container. It will narrow our initial bug search :)

**Comment by salparadise** [comment]
Indeed @bobylito , it works on existing input field. I'm afraid that the documentation statement **CSS Selector or DOMElement to _insert_ the widget** is a bit misleading in that case. :)

**Comment by vvo** [comment]
Fixing this right now, thanks @salparadise

**Comment by vvo** [comment]
Released as part of 1.1.3, let us know @salparadise

**Comment by salparadise** [comment]
It works now, @vvo . Thanks.
